sl_patches/mercury_combined.patch

```diff
@@ -5,7 +5,7 @@ index 08373e3..5c55d7c 100644
 @@ -147,10 +147,17 @@ na_info_parse(const char *info_string, struct na_info **na_info_ptr)
          NA_CHECK_ERROR(na_info->class_name == NULL, error, ret, NA_NOMEM_ERROR,
              "Could not duplicate NA info class name");
- 
+
```

Comment:
I don't think you want to do this.  It's a shame that linting complained about the content of a patch, but I think we have to just accept it and not modify the patch.  Probably linting needs to be modified to skip files in `sl_patches/`.  Please feel free to open a ticket against `scons_local` for that.

Reply:
In general I agree that we should skip files in sl_patches/, however this particular patch was manually produced by me, and in the first iteration I ended up adding extra whitespaces that shouldnt be there to start with.